### PR TITLE
Copy editing: Unit 1

### PR DESCRIPTION
**Pull request by ericoulster**
As I read unit 1 for the first time, I made a point to look for typos & grammatical errors. In a couple of cases, I trimmed down the word choices in select sentences for brevity.  I made sure to retain the same context and meaning for all edits, and focused exclusively on sentence structure, spelling & syntax.

### CODE REVIEW

File: chapters/en/unit1/chapter1/applications.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+In today's world, computer vision systems perform ever more challenging tasks. Some of which are even difficult for humans to do. Let's consider India. Do you know India has the highest number of registered two-wheelers in the world? With that many drivers, some of them they forget to use a helmet. This is dangerous practice, and can cause severe injuries. To address this issue, the government of India, in collaboration with other institutions, developed a computer vision system that automatically catches riders without a helmet and their license plate. The system imposes strict fines against them to discourage people from breaking the law.
```
Comment (mmhamdy) [inline chapters/en/unit1/chapter1/applications.mdx:3]:
```suggestion
In today's world, computer vision systems perform ever more challenging tasks. Some of which are even difficult for humans to do. Let's consider India. Do you know India has the highest number of registered two-wheelers in the world? With that many drivers, some of them forget to use a helmet. This is dangerous practice, and can cause severe injuries. To address this issue, the government of India, in collaboration with other institutions, developed a computer vision system that automatically catches riders without a helmet and their license plate. The system imposes strict fines against them to discourage people from breaking the law.

```

**Review by mmhamdy — COMMENTED** [review]
Thanks! looks good. But I'm not sure about some of the stylistic changes. What do you think @bellabf?

File: chapters/en/unit1/chapter1/applications.mdx [hunk 2]
```diff
@@ -69,19 +70,19 @@ Computer Vision systems face a multitude of challenges that arise from the compl
...
 
-The London Hospital survival predictor is a great example of that. It was created in 1972 and its job was to predict whether patients would recover from a coma. It had a dial that either indictated "survive" or "irreversible brain damage". It was the first applications of patter recognition. Most surprisingly, it was one of the early forms of neural networks. Even at its birth, this raised concerned and doctors were advise to not take its input solely and the machine was never used to remove patient's life support.
+The London Hospital survival predictor is a great example of that. It was created in 1972 and its job was to predict whether patients would recover from a coma. It had a dial that either indicated "survive" or "irreversible brain damage". It was the first applications of pattern recognition. It was one of the early forms of artificial neural networks. During its creation, this raised concerns and doctors were advised not to make judgements solely from the predictor's judgements, and the machine was never used to remove patient's life support.
```
[the remark this answers is not in the document]
Comment (ericoulster) [reply chapters/en/unit1/chapter1/applications.mdx:74]: I agree with the change, it flows better. Worth noting there is a typo. "...pattern recognition an artificial neural networks"

Either pattern recognition _in_ artificial neural networks, or pattern recognition _and_ artificial neural networks.

[the remark this answers is not in the document]
Comment (bellabf) [reply chapters/en/unit1/chapter1/applications.mdx:74]: Fix it :) you should be able to commit it now

File: chapters/en/unit1/chapter1/applications.mdx [hunk 3]
```diff
@@ -69,19 +70,19 @@ Computer Vision systems face a multitude of challenges that arise from the compl
...
 
-Life has changed between then and now. We lived in a global, digital society. The ethical considerations of a model must keep considered in these lenses too. Today, we talk a lot of about buildin fair models. Fairness, in this context, refers to a property of a model to behave in a equitative manner, without target discrimination or unfair bias against a group of a individual. Where bias refers to the inclination for or against a person or group. This is tricky in practice.
+Since then, the world has become a more digital, interconnected place. The ethical considerations of a model must consider fairness and bias in a global perspective. Fairness, in this context, refers to a property of a model to behave in a equitative manner, without target discrimination or unfair bias against a group or individual. Bias refers to the inclination for or against a person or group. Considering fairness and bias can be tricky in practice.
```
[the remark this answers is not in the document]
Comment (ericoulster) [reply chapters/en/unit1/chapter1/applications.mdx:76]: I agree with the changes pacing wise, though _does_ in the change is grammatically incorrect:

I would either word things:
"The world has become a more digital, interconnected place and the ethical considerations for a model must reflect that."

or closer to the original:
The world has become a more digital, interconnected place, as are the ethical considerations for a model.

[the remark this answers is not in the document]
Comment (bellabf) [reply chapters/en/unit1/chapter1/applications.mdx:76]: I think the first one is better. I also updated the commit.

**Review by bellabf — APPROVED** [review]
Thank you so much! Really good job copy editing this file.

File: chapters/en/unit1/chapter1/applications.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (bellabf) [reply chapters/en/unit1/chapter1/applications.mdx]: Thank you!

**Comment by ericoulster** [comment]
> Thank you so much! Really good job copy editing this file.

You're welcome- and thank you for the course! Just finished unit 2 and am already learning from it!

**Review by johko — APPROVED** [review]
LGTM :+1:

**Pull request merged**